<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1561</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;  北斗GPS定位解析系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>控制面板</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="button_startDevice">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 0, 127);</string>
                  </property>
                  <property name="text">
                   <string>开启设备</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_stopDevice">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>关闭设备</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>纬度：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_latitude"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>经度：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_longitude"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>速度：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_speed"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>高度：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_altitude"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>时间：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_UTC"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="labelMouseLongitude">
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="textFormat">
                 <enum>Qt::PlainText</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelMouseLatitude">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="textFormat">
                 <enum>Qt::PlainText</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
           <zorder></zorder>
           <zorder></zorder>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>GPS接收信息</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);
font: 7pt &quot;宋体&quot;;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>地图显示</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonSatelliteMap">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
                <weight>9</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 255);
font: 75 10pt &quot;黑体&quot;;</string>
              </property>
              <property name="text">
               <string>卫星图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonStreetMap">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
                <weight>9</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 0, 0);
font: 75 10pt &quot;黑体&quot;;</string>
              </property>
              <property name="text">
               <string>街道图</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWebView" name="webView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
